docs/installation/tks/Installing_TKS_Clone.md

```diff
@@ -0,0 +1,122 @@
+Installing TKS Clone
+====================
+
+Overview
+--------
+
+This page describes the process to install a TKS subsystem as a clone of an existing TKS subsystem.
+
+Before beginning with the installation, please ensure that you have configured the directory
+server and added base entries.
+The step is described [here](https://github.com/dogtagpki/pki/wiki/DS-Installation).
+
+Additionally, make sure the FQDN has been [configured](../server/FQDN_Configuration.adoc) correctly.
+
+Exporting Existing TKS System Certificates
+------------------------------------------
+
+On the existing system, export the TKS system certificates (including the certificate chain) into a PKCS #12 file, for example:
+
+```
+$ pki-server tks-clone-prepare \
+    --pkcs12-file tks-certs.p12 \
+    --pkcs12-password Secret.123
+```
+
+If necessary, third-party certificates (e.g. trust anchors) can be added into the same PKCS #12 file with the following command:
+
+```
+$ pki -d /etc/pki/pki-tomcat/alias -f /etc/pki/pki-tomcat/password.conf \
+    pkcs12-cert-import <nickname> \
+    --pkcs12-file tks-certs.p12 \
+    --pkcs12-password Secret.123 \
+    --append
+```
+
+TKS Subsystem Installation
+--------------------------
+
+Prepare a deployment configuration (e.g. `tks-clone.cfg`) to deploy TKS subsystem clone.
+By default the subsystem will be deployed into a Tomcat instance called `pki-tomcat`.
+
+A sample deployment configuration is available at [/usr/share/pki/server/examples/installation/tks-clone.cfg](../../../base/server/examples/installation/tks-clone.cfg).
+It assumes that the primary CA and TKS are running at https://primary.example.com:8443,
+the CA signing certificate has been exported into `ca_signing.crt`,
+the admin certificate and key have been exported into `ca_admin_cert.p12`,
+and the password for this file has been exported into `pkcs12_password.conf`.
+See [Installing CA](../ca/Installing_CA.md) for details.
+
+To start the installation execute the following command:
+
+```
+$ pkispawn -f tks-clone.cfg -s TKS
+```
+
+TKS System Certificates
+-----------------------
+
+After installation the CA signing certificate and the TKS system certificates and keys will be stored
+in the server NSS database (i.e. `/etc/pki/pki-tomcat/alias`):
+
```

Comment:
for peace of mind to people, maybe add ", where the sslserver cert is replaced with a newly issued certificate to replace the original." (assuming that's what pkispawn would do)

Reply:
I've updated the doc to say that the `pki-server tks-clone-prepare` will only export the subsystem and audit signing certs, and after installation a new SSL server cert will be created for the new instance.